metaserver/src/main/scala/scala/meta/languageserver/Main.scala

```diff
@@ -13,10 +13,11 @@ import org.langmeta.internal.io.PathIO
 object Main extends LazyLogging {
   def main(args: Array[String]): Unit = {
     val cwd = PathIO.workingDirectory
-    val config = ServerConfig(cwd)
-    Files.createDirectories(config.configDir.toNIO)
-    val out = new PrintStream(new FileOutputStream(config.logFile))
-    val err = new PrintStream(new FileOutputStream(config.logFile))
+    val configDir = cwd.resolve(".metaserver").toNIO
+    val logFile = configDir.resolve("metaserver.log").toFile
+    Files.createDirectories(configDir)
+    val out = new PrintStream(new FileOutputStream(logFile))
+    val err = new PrintStream(new FileOutputStream(logFile))
```

Comment:
Is this ok to have two streams to the same file?  I have never tried it,  but would have expected err and out to point to the same stream, instead of having two streams point to the same file.

Reply:
This was already here, but it's a good question. I guess it's ok since it never gave us problems.